support-frontend/assets/components/subscriptionCheckouts/address/addressFields.tsx

```diff
@@ -184,10 +189,47 @@ export function AddressFields({ scope, ...props }: PropTypes): JSX.Element {
 				data-qm-masking="blocklist"
 				label="Country"
 				value={props.country}
-				onChange={(e) => {
-					const isoCountry = Country.fromString(e.target.value);
-					if (isoCountry) {
-						props.setCountry(isoCountry);
+				onChange={(event) => {
+					const selectedCountry = Country.fromString(event.target.value);
+
+					if (selectedCountry && countryGroupId) {
```

Comment:
These transforms are a little verbose, but necessary - I am transforming from `CountryGroupId` then looking up the `CountryGroupId` of a `Country` and then redirecting if the `current` and `selected` country groups don't match.